pkg/reaper/nodereaper/helpers.go

```diff
@@ -101,11 +101,11 @@ func runCommandWithContext(call string, args []string, timeoutSeconds int64) (st
 	return string(out), nil
 }
 
-func (ctx *ReaperContext) uncordonNode(name string, dryRun bool) error {
+func (ctx *ReaperContext) uncordonNode(name string, dryRun bool, ingoreDrainFailure bool) error {
 	uncordonArgs := []string{"uncordon", name}
 	uncordonCommand := ctx.KubectlLocalPath
-	if dryRun {
-		log.Warnf("dry run is on, instance not uncordoned")
+	if dryRun || ingoreDrainFailure {
+		log.Warnf("dry run / ignore drain failure is on, instance not uncordoned")
```

Comment:
`ingore` should be `ignore`

Reply:
Should we also output which instance was left `cordoned`?